src/img2cmap/convert.py

```diff
@@ -133,3 +133,11 @@ def resize(self, size=(512, 512)):
             resampling_technique = Image.LANCZOS
 
         self.image.thumbnail(size, resampling_technique)
+
+    def remove_transparent(self):
+        """Removes the transparency from the pixels in an image.
```

Comment:
Should we change this to something like: "Removes the transparent pixels from an image array"?

Reply:
Aye captain, pushed it up